### PR TITLE
#161348750  Setup SASS processing on the client

### PR DESCRIPTION
**Pull request by Sulaiman-Mozes**
#### What does this PR do?
 - Setup SASS processing on the client

#### Description of Task to be completed?
 - Add node-sass
 - Add sass-loader to translate sass into css

#### How should this be manually tested?
 - Install dependencies using `npm install`
 - Running the app with `npm start` should display a `React here` message that was styled using a sass file.

#### What are the relevant pivotal tracker stories?
[#161348750](https://www.pivotaltracker.com/story/show/161348750)

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+h1 {
+  color: $green;
+  font-weight: bold;
+  font-size: 3.5rem;
+
+  &:after {
+    content: '👋';
+    display: inline-block;
+    transform: rotate(-20deg);
+    animation: wave .75s infinite linear alternate;
```
Comment (RachaelNantale) [inline src/App.scss]: Add newline at the end of file

**Review by RachaelNantale — CHANGES_REQUESTED** [review]
Please make the necessary changes

File: src/App.scss (hunk 1 above)
Comment (RachaelNantale) [inline src/App.scss]: Add a new line at the end of the file

**Review by RachaelNantale — APPROVED** [review]

**Review by dnuwa — APPROVED** [review]

**Review by brucemakallan — APPROVED** [review]
All good. Thanks

**Pull request merged**